Sources/AuthFoundation/JWT/ClaimContainer.swift

```diff
@@ -16,13 +16,13 @@ import Foundation
 ///
 /// > Note: This does not apply to JWT, which while it contains claims, it has a different format which includes headers and signatures.
 public protocol ClaimContainer: JSONDecodable {
-    var rawValue: [String:Any] { get }
+    var payload: [String:Any] { get }
```

Comment:
```suggestion
    var payload: [String: Any] { get }
```

Reply:
I have a ticket later on in the process to implement swift lint.  :-D